index.d.ts

```diff
@@ -24,12 +24,12 @@ export namespace GoogleSigninButton {
   enum Size {
     Standard,
     Wide,
-    Icon
+    Icon,
   }
 
   enum Color {
     Light,
-    Dark
+    Dark,
```

Comment:
Not sure if you are abiding by a style guide that includes trailing commas, but they aren't my preference :)
(several more of these in the .flow file)

Reply:
I used the same config that is used to format the JS files, so this we're consistent